_pages/fr/help/specific-agencies/overview.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: help
+title: Aide aux agences spécifiques
+category: specific-agencies
+order: 0
+redirect_from:
+  - /help/specific-agencies/
```

Comment:
are `redirect_from` urls localized?

my guess is these shouldn't be here? It seems unexpected to me to redirect from a plain (english) URL to the French one

Reply:
They should be localized. I fix this in #588